src/design-tokens/tier-1-definitions/typography-presets.css

```diff
@@ -133,7 +133,7 @@
   font-family: var(--eds-font-family-primary) $important;
   font-weight: var(--eds-font-weight-medium) $important;
   font-size: var(--eds-font-size-14) $important;
-  line-height: 1.57 $important;
```

Comment:
This particular preset is very confused. In code, it is always 14/22. In design, it is sometimes 14/22 (14/24 for regular, and 14/22 for light and bold). In storybook it is always 14/24 (but 006-light is missing from the page). currently trying to clarify

Reply:
Fixing up to be based on 14/22 across the board